.travis.yml

```diff
@@ -5,6 +5,12 @@ julia:
     - nightly
 notifications:
     email: false
+
+addons:
+  apt:
+    packages:
+    - hdf5-tools
+
```

Comment:
The problem is that indirect dependencies require hdf5 and due to changes in Travis infrastructure, the sudo call to install this fails.